README.md

```diff
@@ -38,6 +38,18 @@ The design process for changes to M3 is loosely modeled on the [proposal process
 
 - The author (and/or other contributors) do the work as described by the "Implementation" section of the proposal.
 
+## Approval 
+
+The proposal requires approval from all Technical Steering Committee (the “TSC”) members.
+
+The current TSC members are:
```

Comment:
can you sort these alphabetically?

also, should we add organizations? e.g. `@mway (Uber)`, `@robskillington (Chronosphere)`?